src/sharedData/components/SharedFileIcon.js

```diff
@@ -39,13 +40,19 @@ const ICON_FILES = {
   png: 'png.png',
 };
 
-const SharedFileIcon = ({ resourceType }) => {
+const SharedFileIcon = props => {
+  const { resourceType, styleProps, fallbackStyleProps } = props;
```

Comment:
nitpick: You can do the destructuring in one line if you want
`const SharedFileIcon = ({ resourceType, styleProps, fallbackStyleProps }) => { 
`
and not need `const { resourceType, styleProps, fallbackStyleProps } = props;`

like how it was before with `({ resourceType })`